lib/check-started.js

```diff
@@ -1,31 +1,30 @@
 module.exports = checkStarted;
 
-var request = require('request').defaults({json: true});
+const got = require('got');
 var statusUrl = require('./get-selenium-status-url.js');
 
 function checkStarted(seleniumArgs, cb) {
-  var retries = 0;
-  var hub = statusUrl.getSeleniumStatusUrl(seleniumArgs);
-  // server has one minute to start
-  var retryInterval = 200;
-  var maxRetries = 60 * 1000 / retryInterval;
-
-  function hasStarted() {
-    retries++;
-
-    if (retries > maxRetries) {
-      cb(new Error('Unable to connect to selenium'));
-      return;
+  const seleniumStatusUrl = statusUrl.getSeleniumStatusUrl(seleniumArgs);
+  const options = {
+    timeout: 1000,
+    retry: {
+      limit:300,
+      calculateDelay: ({attemptCount, retryOptions, error, computedValue}) => {
+        // server has one minute to start
+        // retry to connect every 200 ms up until the limit of 300 attempts is reached
+        return 200
+      }
     }
-
-    request(hub, function (err, res) {
-      if (err || res.statusCode !== 200) {
-        setTimeout(hasStarted, retryInterval);
-        return;
+  };
+  (async () => {
+      try {
+          const res = await got(seleniumStatusUrl, options);
+          if(res.statusCode !== 200) {
```

Comment:
I guess if status code is not 200 for throws an error so it cannot be ever reached